src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -37,6 +37,38 @@ void regclass_Tensor(py::module m) {
                 :type shared_memory: bool
             )");
 
+    cls.def(py::init([](int64_t data_ptr, const ov::Shape& shape, const ov::element::Type& ov_type) {
```

Comment:
can `data_ptr` be a `void *` here?
if not, I suggest to use `uint64_t`

Reply:
Same in [line](https://github.com/openvinotoolkit/openvino/pull/27725/files#diff-622a251f061d9a537d2f4a087dc8e5666ce22fe246abc7cf69c0b31e5abb9870R493)